docs/advanced_usage.md

```diff
@@ -418,6 +423,92 @@ Insurances/                             # Insurances
     Defining a storage path is optional. If no storage path is defined for a
     document, the global [`PAPERLESS_FILENAME_FORMAT`](configuration.md#PAPERLESS_FILENAME_FORMAT) is applied.
 
+### Filename Templates {#filename-templates}
+
+The filename formatting uses [Jinja templates](https://jinja.palletsprojects.com/en/3.1.x/templates/) to build the filename.
```

Comment:
Shouldn't this be Django templates?

Reply:
Look at the commits

Reply:
Alright, now it makes sense. Probably got confused by your answer to my issue https://github.com/paperless-ngx/paperless-ngx/discussions/5390 😅